tests/foreman/cli/test_repository.py

```diff
@@ -201,6 +203,154 @@ def test_positive_create_with_auth_yum_repo(self):
                 self.assertEqual(new_repo['content-type'], u'yum')
 
     @tier1
+    def test_positive_create_with_download_policy(self):
+        """Create YUM repositories with available download policies
+
+        @id: ffb386e6-c360-4d4b-a324-ccc21768b4f8
+
+        @Assert: YUM repository with a download policy is created
+        """
+        for policy in DOWNLOAD_POLICIES:
```

Comment:
One minor downside to not using `subTest` is that if one of the download policies fails, the entire test will fail and we won't try all permutations.